website/content/en/news/_index.adoc

```diff
@@ -13,14 +13,13 @@ loadNews: false
 * link:../press/[Press articles]: FreeBSD appearing in the regular press.
 * link:../status/[Status reports]: FreeBSD development status reports.
 
-== Podcasts
+== Podcast
 
-* https://www.bsdnow.tv/[BSD Now]: BSD Now is a weekly video podcast which covers the latest news, interviews and tutorials from the world of BSD.
-* https://bsdtalk.blogspot.com/[bsdtalk]: bsdtalk is an audio podcast with interesting interviews of people in the BSD community.
+* https://www.bsdnow.tv/[BSD Now] (weekly) covers the latest news, interviews and tutorials in the world of BSD.
 
 == Other sites
 
-* http://www.daemonforums.org/[Daemon Forums]: Active online forums and news community site dedicated to FreeBSD and other BSDs.
-* https://www.freebsddiary.org/[FreeBSD Diary]: One man's record of his trials and triumphs with FreeBSD.
+* https://dan.langille.org/[Dan Langille's Other Diary]: successor to _The FreeBSD Diary_, which provided practical examples between 1988 and 2012. Dan is, https://www.langille.org/[amongst other things], a committer to the FreeBSD ports tree. 
```

Comment:
@dlangille, for the purpose of <https://www.freebsd.org/news/>, would you agree with my use of the word _successor_?

Reply:
Whilst there's no longer a direct link to (old) _The FreeBSD Diary_, readers should easily find their way – if required – via the _amongst other things_ link.

Reply:
Sure. Thanks.